AmazonChimeSDK/AmazonChimeSDK/audiovideo/AudioVideoConfiguration.swift

```diff
@@ -10,47 +10,101 @@ import Foundation
+                  enableAudioRedundancy: enableAudioRedundancy,
+                  videoMaxResolution: Self.defaultVideoMaxResolution)
     }
 
     convenience public init(videoMaxResolution: VideoResolution) {
-        self.init(audioMode: .stereo48K, callKitEnabled: false, enableAudioRedundancy: true, videoMaxResolution: videoMaxResolution)
+        self.init(audioMode: Self.defaultAudioMode,
+                  audioDeviceCapabilities: Self.defaultAudioDeviceCapabilities,
+                  callKitEnabled: Self.defaultCallKitEnabled,
+                  enableAudioRedundancy: Self.defaultEnableAudioRedundancy,
+                  videoMaxResolution: videoMaxResolution)
     }
 
     convenience public init(audioMode: AudioMode, callKitEnabled: Bool) {
-        self.init(audioMode: audioMode, callKitEnabled: callKitEnabled, enableAudioRedundancy: true, videoMaxResolution: VideoResolution.videoResolutionHD)
+        self.init(audioMode: audioMode,
+                  audioDeviceCapabilities: Self.defaultAudioDeviceCapabilities,
+                  callKitEnabled: callKitEnabled,
+                  enableAudioRedundancy: Self.defaultEnableAudioRedundancy,
+                  videoMaxResolution: Self.defaultVideoMaxResolution)
     }
 
     convenience public init(audioMode: AudioMode, callKitEnabled: Bool, enableAudioRedundancy: Bool) {
-        self.init(audioMode: audioMode, callKitEnabled: callKitEnabled, enableAudioRedundancy: enableAudioRedundancy, videoMaxResolution: VideoResolution.videoResolutionHD)
+        self.init(audioMode: audioMode,
+                  audioDeviceCapabilities: Self.defaultAudioDeviceCapabilities,
+                  callKitEnabled: callKitEnabled,
+                  enableAudioRedundancy: enableAudioRedundancy,
+                  videoMaxResolution: Self.defaultVideoMaxResolution)
     }
 
-    public init(audioMode: AudioMode, callKitEnabled: Bool, enableAudioRedundancy: Bool, videoMaxResolution: VideoResolution) {
+    public init(audioMode: AudioMode = AudioVideoConfiguration.defaultAudioMode,
+                audioDeviceCapabilities: AudioDeviceCapabilities = AudioVideoConfiguration.defaultAudioDeviceCapabilities,
+                callKitEnabled: Bool = AudioVideoConfiguration.defaultCallKitEnabled,
+                enableAudioRedundancy: Bool = AudioVideoConfiguration.defaultEnableAudioRedundancy,
+                videoMaxResolution: VideoResolution = AudioVideoConfiguration.defaultVideoMaxResolution) {
         self.audioMode = audioMode
+        self.audioDeviceCapabilities = audioDeviceCapabilities
         self.callKitEnabled = callKitEnabled
         self.enableAudioRedundancy = enableAudioRedundancy
         self.videoMaxResolution = videoMaxResolution
     }
 
     override public var description: String {
-        return "audioMode: \(self.audioMode.description), callKitEnabled: \(self.callKitEnabled), enableAudioRedundancy: \(self.enableAudioRedundancy)"
```

Comment:
Looks like we missed `videoMaxResolution` previously, will be very appreciated if you can add it. 🙇

Reply:
Added in latest commit